tcpdf.php

```
@@ -7710,7 +7710,15 @@ public function _destroy($destroyall=false, $preserve_objcopy=false) {
 		}
 		if ($destroyall AND !$preserve_objcopy) {
 			self::$cleaned_ids[$this->file_id] = true;
-			// remove all temporary files
+			// first: remove all imagekeys files
+			if (isset($this->imagekeys)) {
+				foreach($this->imagekeys as $file) {
+					if (strpos($file, K_PATH_CACHE) === 0 && file_exists($file)) {
+						unlink($file);
```

Note: the STFU operator was removed, I assume it was intentional

---

Yes, I understand that was used initially to hide the original error when try to unlink an unlinked previously file